src/Controls/src/Core/Label/Label.iOS.cs

```diff
@@ -95,6 +95,11 @@ static void MapFormatting(ILabelHandler handler, Label label)
 		{
 			handler.UpdateValue(nameof(ILabel.TextColor));
 			handler.UpdateValue(nameof(ILabel.Font));
+
+			if (!IsPlainText(label))
+				return;
+
+			LabelHandler.MapFormatting(handler, label);
```

Comment:
Could you include a test to validate it and avoid in the future?

Something like:

```
[Fact(DisplayName = "Using CharacterSpacing with LineHeight and TextDecorations works Correctly")]
public async Task CharacterSpacingWithLineHeightWithTextDecorationsWorksCorrectly()
{
	var expectedCharacterSpacing = 5;
	var expectedLineHeight = 1.5d;
	var expectedTextDecorations = TextDecorations.Underline;

	var label = new LabelStub
	{
		Text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
		CharacterSpacing = expectedCharacterSpacing,
		LineHeight = expectedLineHeight,
		TextDecorations = expectedTextDecorations
	};

	var handler = await CreateHandlerAsync(label);

	var actualCharacterSpacing = await InvokeOnMainThreadAsync(() => GetNativeCharacterSpacing(handler));
	Assert.Equal(expectedCharacterSpacing, actualCharacterSpacing);

	var actualLineHeight = await InvokeOnMainThreadAsync(() => GetNativeLineHeight(handler));
	Assert.Equal(expectedLineHeight, actualLineHeight);

	var attributedText = await InvokeOnMainThreadAsync(() => GetAttributedText(handler));
	attributedText.AssertHasUnderline();
}
```

In Core.DeviceTests.

Reply:
Hi, @jsuarezruiz 
Thank you for showing me how to write tests.
I would like to write a test case.

Reply:
Hi, @jsuarezruiz 
I added the necessary code to AssersionExtension.iOS.cs and LabelTest.iOS.cs to write the test case.
Test cases are PlainText, FormattedText, and HTML patterns.

Reply:
Awesome, it looks nice.